src/main/java/nebula/plugin/metrics/MetricsPlugin.java

```diff
@@ -130,11 +137,20 @@ public void execute(Project project) {
 
     }
 
+    public void addCustomDispatcher(MetricsDispatcher dispatcher) {
```

Comment:
`add` might invoke a thought that there might be multiple dispatchers. Should we just use the existing setter and make it public? What do you think?

Reply:
I like it! https://github.com/nebula-plugins/gradle-metrics-plugin/pull/41/commits/4ba39dd8b04ba48338ab443afeb815a1a4ff09c7 has the change